### ISSUE TITLE
No MSI in releases

### ISSUE THREAD

**Issue by mrexodia** [mesonbuild/meson#9142]
**Describe the bug**

According to the [documentation](https://mesonbuild.com/SimpleStart.html#windows1) there should be an MSI at https://github.com/mesonbuild/meson/releases, but there isn't. This makes it kind of hard to follow the quick start guide for a new user.

**Comment by jpakkane** [comment]
Thanks for reporting. This was forgotten by accident and has now been added.

**Comment by mensinda** [comment]
@jpakkane What is the process for creating these MSI files? We should be able to automate this with GitHub actions right?

**Comment by mrexodia** [comment]
Thanks!
